engines/python/setup/djl_python/deepspeed.py

```diff
@@ -357,6 +357,11 @@ def create_model_pipeline(self):
             self.ds_config["replace_with_kernel_inject"] = False
         state_dict_mmap = {}
         if "checkpoint" in self.ds_config:
```

Comment:
should we just disable checkpoint option entirely, given we already have cpu loading optimization?

Reply:
It would be a feature regression. Is anyone using it? We can unify things pretty easily in future version so that checkpoint loading works in unison with all other features, we just didn't get to that yet.